BusRoutes/Model/Route.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace BusRoutes.Model
+{
+	public class Route
+	{
+		int id;
+
+		[JsonProperty("id")]
+		public int Id {
+			get {
```

Comment:
You should be consistent with the position of the curly braces: either use them in the same line or in the next, avoid mixing them up, specially in the same file ;-)

Keep in mind that the Mono/.NET standard is to break the line before the curly brace, but whatever you choose, be consistent.